mws/sitesmanagement/models.py

```diff
@@ -623,6 +623,23 @@ def __unicode__(self):
         return self.name
 
 
+    def domains(self, global_only=None):
```

Comment:
I think the "tri-state bool" (None, True, False) is a little non-ovbious here. Could we define some constants. E.g.

```python
ALL_HOSTS = 'all'
GLOBAL_HOSTS = 'global'
PRIVATE_AND_GLOBAL_HOSTS = 'private_global'

def domains(self, subset=ALL_HOSTS):
    if subset is ALL_HOSTS:
        # ....
    elif subset is GLOBAL_HOSTS:
        # ....
    elif subset is PRIVATE_AND_GLOBAL_HOSTS:
        # ....
    else:
        raise ValueError('Unknown subset type: %r' % (subset,))
```

Reply:
I did think it was off a bit. one thing that becomes less explicit with this indirection is the fact that the service hostname is removed from global names but not the list of private and global names (yet).